fedmsg_meta_fedora_infrastructure/tests/pkgdb.py

```diff
@@ -366,7 +366,7 @@ class LegacyTestPkgdbBranchClone(Base):
     }
 
 
-class TestPkgdbCritpathUpdate(Base):
+class TestLegacyPkgdbCritpathUpdate(Base):
```

Comment:
There is still a pkgdb2.package.critpatch.update message: https://github.com/fedora-infra/pkgdb2/blob/master/pkgdb2/lib/utils.py#L264